client/src/components/RatingsComponents/ReviewsList/NewReviewForm.jsx

```diff
@@ -0,0 +1,161 @@
+import React, {useState, useEffect} from 'react';
+import {token} from '../../../../../config.js';
+import Axios from 'axios';
+
+function NewReviewForm ({factors}) {
+  const factorGrades = {
+    'Size': ['A size too small', '1/2 size too small', 'Perfect', '1/2 size too big', 'A size too wide'],
+    'Width': ['Too narrow', 'Slightly narrow', 'Perfect', 'Slightly wide', 'Too Wide'],
+    'Comfort': ['Uncomfortable', 'Slightly uncomfortable', 'Ok', 'Comfortable', 'Perfect'],
+    'Quality': ['Poor', 'Below average', 'What I expected', 'Pretty great', 'Perfect'],
+    'Length': ['Runs short', 'Runs slightly short', 'Perfect', 'Runs slightly long', 'Runs long'],
+    'Fit': ['Runs tight', 'Runs slightly tight', 'Perfect', 'Runs slightly long', 'Runs long']
+  };
+
+  console.log(factors);
+  // const createFactors = () => {
+
+  // }
```

Comment:
Is this a new function you're working on? If not, just remove

Reply:
i might pull the functionality of this function out onto its own separate file in a refactor but for now its working and implemented in the form itself so ill just delete it for now.